task_Dev_2/calculator/Calculator.java

```diff
@@ -0,0 +1,80 @@
+package task_Dev_2.calculator;
+
+import java.util.*;
+
+/**
+ * Calculate expression stored in list.
+ */
+class Calculator {
+  private static final String DEV = "/";
+  private static final String PROD = "*";
+  private static final String DIF = "-";
+  private static final String SUM = "+";
+  
+  /**
+   * Calculate results by priority of operations. Firstly they are divided and multiplication, 
+   * then sum and substraction.
+   * Returns nothing. 
+   * @param exprArr - list of string elements
+   */
+  public void calculateResult(List<String> exprArr) {
+	System.out.println("Size: " + exprArr.size());
+	try {
+	  reWriteList(PROD, DEV, exprArr);
+	  reWriteList(SUM, DIF, exprArr);
+	} catch (NumberFormatException ex) {
+	  System.out.println("Sorry! Incorrect number format!");
```

Comment:
make message less emotional but more detailed - print which number is incorrect and remove all these !!